examples/wireguard.yml

```diff
@@ -68,6 +69,7 @@ files:
   - path: etc/wireguard/wg1.conf
     contents: |
       [Interface]
+      Address = 192.168.2.2/24
       PrivateKey = SCGCKDuTm4PMOw+LXdK/2s8mxnv145QHOohKRq3vc2A=
       ListenPort = 51821
```

Comment:
By the way, if you don't specify ListenPort, the kernel will choose one for you at random, which could be desirable for some.